windows/src/developer/inst/copydev.in

```diff
@@ -22,19 +22,19 @@ KEYMAN_DEVELOPER_TEMPLATES_ROOT=$(ROOT)\src\developer\kmconvert\data
 KEYMAN_MODELCOMPILER_ROOT=$(KEYMAN_ROOT)\developer\js
 
 copykmdev: makeinstaller
-    -mkdir $(ROOT)\release\$VERSION
-    copy /Y $(ROOT)\src\developer\inst\keymandeveloper.msi $(ROOT)\release\$VERSION\keymandeveloper.msi
-    copy /Y $(ROOT)\src\developer\inst\keymandeveloper-$VERSION.exe $(ROOT)\release\$VERSION\keymandeveloper-$VERSION.exe
+    -mkdir $(ROOT)\release\$Version
+    copy /Y $(ROOT)\src\developer\inst\keymandeveloper.msi $(ROOT)\release\$Version\keymandeveloper.msi
+    copy /Y $(ROOT)\src\developer\inst\keymandeveloper-$VersionWithTag.exe $(ROOT)\release\$Version\keymandeveloper-$VersionWithTag.exe
 
 test-releaseexists:
-    if exist $(ROOT)\release\$VERSION\keymandeveloper*.msi echo. & echo Release $VERSION already exists. Delete it or update src\version.txt and try again & exit 1
+    if exist $(ROOT)\release\$Version\keymandeveloper*.msi echo. & echo Release $Version already exists. Delete it or update src\version.txt and try again & exit 1
```

Comment:
Is `src\version.txt` now `..\..\VERSION.md`?

Reply:
Yes...